nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,15 @@
+# URL of map resource
+# Can be an absolute path to a file: file:///path/to/maps/floor1.yaml
+# Or, relative to a ROS package: package://my_ros_package/maps/floor2.yaml
+string map_url
+---
+# Result code defintions
+uint8 RESULT_SUCCESS=0
```

Comment:
I'd prefer if these were just `SUCCESS` etc without the `RESULT_` prefix since in use it would be `nav_msgs::msg::LoadMap::Result::RESULT_MAP_DOES_NOT_EXIST` and we see `result` twice in a row.

Reply:
Actually, its not:
`nav_msgs::msg::LoadMap::Result::RESULT_MAP_DOES_NOT_EXIST`
It's
`nav_msgs::msg::LoadMap::Response::RESULT_MAP_DOES_NOT_EXIST`

Subtle difference.

Reply:
ah, yes sorry. 

Its not something I care deeply about, just a comment that it seems related enough with the namespace the additional wording just uses linespace. If no one else agrees, I'm certainly not going to push it.